R/seq_linter.R

```diff
@@ -42,6 +47,11 @@
 #'   linters = seq_linter()
 #' )
 #'
+#' lint(
+#'   text = "sapply(x, seq_len)",
```

Comment:
Shouldn't this example use `sequence()`?

Reply:
No preference really. I'm always a bit confused by negative examples because they could be almost anything really :shrug:

Happy to hear any guidelines on this or just follow your instinct if you think that's better.

Reply:
We are trying to follow a one-to-one mapping between examples with-to-without lints in such a way that the without version follows the provided advice in with version. So if the with version mentions that the user is better off using `sequence()`, it would be preferable to follow that advice in the without version.

Does that make sense? Open to breaking that pattern if you feel strongly about it.

Reply:
Ooooh, yes, that makes sense. I thought they were just false negative but didn't realize they were actually the result of the linter recommendation. I have changed it

Reply:
Thanks!